benchmark/benchmark_view_access.cpp

```diff
@@ -241,28 +233,29 @@ namespace xt
         auto calc_dofval(xt::xtensor<double, 3, L>& elemvec, xt::xtensor<double, 1, L>& dofval)
         {
             dofval.fill(0.0);
-            for (size_t e = 0 ; e < M_NELEM ; ++e)
-                for (size_t m = 0 ; m < M_NNE ; ++m)
-                    for (size_t i = 0 ; i < M_NDIM; ++i)
+            for (size_t e = 0; e < M_NELEM; ++e)
+                for (size_t m = 0; m < M_NNE; ++m)
+                    for (size_t i = 0; i < M_NDIM; ++i)
```

Comment:
I think that we should enforce `{ ... }` everywhere

Reply:
I will try this in a second time.

Reply:
OK!